sdk/core/azure-core-amqp/src/impl/uamqp/amqp/cancellable.cpp

```diff
@@ -3,9 +3,7 @@
 
 #include "azure/core/amqp/internal/cancellable.hpp"
 
-#if ENABLE_UAMQP
```

Comment:
Are we able to remove this now because the entire file is conditionally compiled now?

Reply:
Probably.